src/_common/scroll/scroller/scroller.ts

```diff
@@ -10,65 +9,24 @@ import { AppScrollInviewParent } from '../inview/parent';
 	},
 })
 export default class AppScrollScroller extends Vue {
-	@Prop(Boolean)
-	overlay?: boolean;
+	@Prop(propOptional(Boolean))
```

Comment:
Change these to: `@Prop(propOptional(Boolean, false)) thin: boolean` This way we dont' ahve undefined.